docs/samples/Microsoft.ML.Samples/Dynamic/OnnxTransform.cs

```diff
@@ -34,7 +34,7 @@ public static void Example()
             var mlContext = new MLContext();
             var data = GetTensorData();
             var idv = mlContext.Data.LoadFromEnumerable(data);
-            var pipeline = mlContext.Transforms.ApplyOnnxModel(modelPath, new[] { outputInfo.Key }, new[] { inputInfo.Key });
+            var pipeline = mlContext.Transforms.ApplyOnnxModel(new[] { outputInfo.Key }, new[] { inputInfo.Key }, modelPath);
```

Comment:
>modelPath [](start = 114, length = 9)

I see at line 20 where session is loaded. Does the `ApplyOnnxModel` reload the model? If it does then there needs to be a way to reuse the` session` object created above.

Reply:
That is a good point. Can you file an issue on this? We can do this as we start to fix outstanding issues in the ONNX implementation.

---
In reply to: [268890248](https://github.com/dotnet/machinelearning/pull/3086#discussion_r268890248) [](ancestors = 268890248)